resources/views/livewire/settings/plugins.blade.php

```diff
@@ -47,13 +47,13 @@
         <x-modal name="install" max-width="7xl">
             <x-card :title="__('Install packages')">
                 <div class="flex flex-col gap-4">
-                    <x-features.media.upload-form-object wire:model="file" :multiple="true" accept=".zip,.rar,.7zip">
+                    <x-flux::features.media.upload-form-object wire:model="file" :multiple="true" accept=".zip,.rar,.7zip">
                         <x-slot:footer>
                             <div x-show="$wire.file.stagedFiles.length > 0" x-cloak x-transition class="flex justify-end">
-                                <x-button primary spinner="installUploaded" :label="__('Upload package')" wire:click="installUploaded" wire:confirm.icon.warning="{{ __('wire:confirm.install-uploaded-plugin') }}"/>
+                                <x-button primary spinner="installUploaded" :label="__('Upload package')" wire:click="installUploaded" wire:flux-confirm.icon.warning="{{ __('wire:confirm.install-uploaded-plugin') }}"/>
```

Comment:
remove "flux-". If it is correct to keep it, this must be changed in all other confirm dialogs.